cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -152,6 +152,14 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 	s.SpanReader, err = createSpanReader(logger, ports.QueryGRPC)
 	require.NoError(t, err)
 
+	scrapeCmd := exec.Command("./scripts/scrape-metrics.sh", "-t", storage)
```

Comment:
a) it's a good idea that we can execute scraping from this test, instead of from individual shell scripts
b) if we do it here there is no need to call a shell script, just execute an HTTP GET request from Go and write output to a file.
c) why is scrape dir "../../../.."? I believe the cwd for test is the root of the repository already
d) I would suggest we create a `.gitignore`-d directory in the root like `.metrics` and place all the files there. We don't want to create files that will show up as changes in git.